sdk/core/azure-core/src/main/java/com/azure/core/models/HttpResponseError.java

```diff
@@ -74,7 +74,7 @@ public String getTarget() {
      * @param target the target of this error.
      * @return the updated {@link HttpResponseError} instance.
      */
-    public HttpResponseError setTarget(String target) {
+    HttpResponseError setTarget(String target) {
```

Comment:
Should this class be removed from the public API?

Also, I thought this was going to be `ResponseError` instead of `HttpResponseError`.

Reply:
I moved this to core-experimental as the arch board feedback was not conclusive on the naming. I'll follow up with @JonathanGiles to finalize the name.